editor/code_editor.cpp

```diff
@@ -1604,6 +1600,12 @@ void CodeTextEditor::_error_pressed(const Ref<InputEvent> &p_event) {
 
 void CodeTextEditor::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_READY: {
+			idle->set_wait_time(idle_time);
```

Comment:
`set_error_count` does this
https://github.com/godotengine/godot/blob/2b7ea6223bd0bdaa75706849c9dee1a82b2e2d01/editor/code_editor.cpp#L1629-L1638

So, one way or the other, the wait time will be set appropriately, which makes this line unnecessary. Am I missing something?